website/client/components/tasks/task.vue

```diff
@@ -8,7 +8,7 @@
         .task-control.habit-control(:class="controlClass.up.inner", @click="(isUser && task.up) ? score('up') : null")
           .svg-icon.positive(v-html="icons.positive")
       // Dailies and todos left side control
-      .left-control.d-flex.justify-content-center(v-if="task.type === 'daily' || task.type === 'todo'", :class="controlClass.bg")
+      .left-control.d-flex.justify-content-center(v-if="(task.type === 'daily' || task.type === 'todo') && !task.preventCheck", :class="controlClass.bg")
         .task-control.daily-todo-control(:class="controlClass.inner", @click="isUser ? score(task.completed ? 'down' : 'up') : null")
```

Comment:
the `preventCheck` should be placed here like `isUser && !preventCheck` that we still see checkbox in the preview